bingosync-app/generators/dark_souls_generator.js

```diff
@@ -129,14 +126,14 @@ bingoList[11] = [
 ];
 // Sen's Fortress
 bingoList[12] = [
-  { name: "Obtain Lightning Spear", types: ["items","weapons"] },
+  { name: "Obtain a Spear upgraded with Lightning", types: ["items","weapons"] },
```

Comment:
Lightning Spear is an ambiguous name, as a lightning-infused spear is called **Lightning Spear**, and there is also a miracle called **Lightning Spear**. Given that this is in the Sen's Fortress array, _and_ since earning the miracle **Lightning Spear** is functionally identical to another goal in the generator ("Join The Warriors of Sunlight"), it's safe to assume this means an upgraded spear weapon.